hiring/interviewing.md

```diff
@@ -0,0 +1,112 @@
+Recruitment team will be available to answer any questions you may have throughout the hiring process!
+
+This guide is tailored to our standard interviews for the IC3-IC5 levels on
+[our ladder](/careers/ladder.md#readme). The content of the interviews may change slightly for other levels, but
+the ethos will be the same.
+
+### Phone screens
+
+As a very first step, you’ll speak with one of our recruiters and perhaps an engineering manager. These
+conversations are a way for us to learn more about you, what motivates you, and gauge whether Artsy and the
+opportunity is a good fit for your career goals.
+
+**How to prepare** Be prepared to speak to the work you’ve accomplished on your resume—we may ask you to describe a
+recent project and/or specific technologies you’ve used in the past. Also make sure to come prepared with questions
+to ask your interviewer. Remember, this is also for _you_ to make sure Artsy would be a good fit!
+
+### "On-site" Interview
+
+During your "on-site", you will participate in four 30 minute interviews, and one 60 minute interview:
+
+#### 1. Hiring Manager
+
+This interview is with the hiring manager for the role. The hiring manager will be an Engineering Manager on the
+team, and in _most_ cases, will be your people manager if you are to join Artsy.
+
+This interview focuses on the capabilities described in [our engineering ladder](/careers/ladder.md#readme). You
+will not be asked to whiteboard, but be prepared to talk about past experiences working on teams or projects.
+
+#### 2. Peer (Engineer)
+
+This interview will be with an engineer. We have a semi-fluid team structure, so this person may or may not be on
+the same team as this role.
+
+The goal of this interview is to learn about your experience collaborating with other engineers. Like the hiring
+manager interview, this is non-technical but be prepared to discuss past experiences.
+
+#### 3. Peer (Non-Engineer)
+
+This interview will be with a member of our Product, Design or Data teams.
+
+Similar to the Peer (Engineer) interview above, this is non-technical but you should be prepared to discuss past
+experiences working cross-collaboratively.
+
+#### 4. Technical Fluency
+
+This interview will be with an engineer. The focus of this interview varies for different levels and specialities.
+Most likely, the person you interview with will be at the same (or higher) level than the one you are interviewing
+for and have expertise in the domain.
+
+You will not be asked to code but we will dig into the technical details of projects you've worked on. In general,
+we care more about hearing how you think through problems vs. whether you're familiar with a specific technology.
+
+Some advice: dig into the details! Talk about how you approached problems and why. We like to hear your thought
+process but also understand if you need a few moments to think before answering-- let us know if this is the case!
```

Comment:
Maybe we can encourage 2-way communication here (and also the System Design below). Something like "Don't be afraid of asking clarifying questions!"